### PR TITLE
feature: use official CacheSessionPool from google/cloud-spanner

### PR DESCRIPTION
**Pull request by taka-oyama**

### CODE REVIEW

File: src/Concerns/ManagesSessionPool.php [hunk 1]
```diff
@@ -42,11 +42,7 @@ trait ManagesSessionPool
     public function clearSessionPool(): void
     {
         $sessionPool = $this->getSpannerDatabase()->sessionPool();
-        $operation = $sessionPool?->clear();
-
-        if ($operation instanceof DeleteOperation) {
```
Comment (tyabu12) [inline src/Concerns/ManagesSessionPool.php:47]: Cloud you remove use `Colopl\Spanner\Session\DeleteOperation` ?

File: src/Session/CacheSessionPool.php [hunk 2]
```diff
@@ -1,1048 +0,0 @@
-<?php
```
Comment (tyabu12) [inline src/Session/CacheSessionPool.php:1]: Cloud you also remove excludePaths setting in phpstan.neon?

https://github.com/colopl/laravel-spanner/blob/master/phpstan.neon#L6-L8

File: src/Concerns/ManagesSessionPool.php (hunk 1 above)
Comment (taka-oyama) [reply src/Concerns/ManagesSessionPool.php:47]: done

File: src/Session/CacheSessionPool.php (hunk 2 above)
Comment (taka-oyama) [reply src/Session/CacheSessionPool.php:1]: It has been removed. Thanks.

**Review by tyabu12 — APPROVED** [review]
LGTM

**Pull request merged**